packages/itwinui-react/package.json

```
@@ -64,7 +64,7 @@
     "clean:coverage": "rimraf coverage",
     "clean": "rimraf .turbo && yarn clean:coverage && yarn clean:build && rimraf node_modules",
     "test": "yarn test:types && yarn test:unit",
-    "test:unit": "dotenv -v NODE_OPTIONS=--experimental-vm-modules -- npx jest",
+    "test:unit": "dotenv -v NODE_OPTIONS=--experimental-vm-modules -- npx vitest",
```

The experimental vm flag was for jest and can be removed. Also, `vitest` by default starts in watch mode, so `vitest run` should be used instead.

```suggestion
    "test:unit": "vitest run",
```